### PR TITLE
doc: fix render_upload/3 example

### PR DESCRIPTION
**Pull request by mad42**

### CODE REVIEW

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

File: lib/phoenix_live_view/test/live_view_test.ex [hunk 1]
```diff
@@ -1635,13 +1635,13 @@ defmodule Phoenix.LiveViewTest do
...
-      assert render_upload(avatar, "foo.jpeg") =~ "100%"
+      assert render_upload(avatar, "myfile.jpeg") =~ "100%"
 
   By default, the entire file is chunked to the server, but an optional
   percentage to chunk can be passed to test chunk-by-chunk uploads:
 
-      assert render_upload(avatar, "foo.jpeg", 49) =~ "49%"
-      assert render_upload(avatar, "foo.jpeg", 51) =~ "100%"
+      assert render_upload(avatar, "myfile.jpeg", 49) =~ "49%"
+      assert render_upload(avatar, "myfile.jpeg", 51) =~ "51%"
```
Comment (mcrumm) [inline lib/phoenix_live_view/test/live_view_test.ex:1644]: I believe this last example is incorrect– with `render_upload/3` progress is additive, so two consecutive invocations, with progress values of `49` and `51` respectively, should result in 100% progress:

```elixir
assert render_upload(avatar, "myfile.jpeg", 49) =~ "49%"
assert render_upload(avatar, "myfile.jpeg", 51) =~ "100%"
```

Comment (josevalim) [reply lib/phoenix_live_view/test/live_view_test.ex:1644]: Fixed.

Comment (mad42) [reply lib/phoenix_live_view/test/live_view_test.ex:1644]: Thanks @mcrumm for explaining that, I had a test failing with a value above 100 😄

**Pull request merged**